ias-verify/src/ephemeral_key.rs

```diff
@@ -0,0 +1,32 @@
+use crate::utils::{length_from_raw_data, safe_indexing};
```

Comment:
This code was in `verify_ias_report` but the key was not checked against anything. I extracted it to here, but I do not call it anymore because I don't think extracting the key verifies something valuable.